### DISCUSSION TITLE
遮挡和布局问题

### DISCUSSION THREAD

**Discussion by 540350523** [AlistGo/alist#1104] [General]
![1](https://user-images.githubusercontent.com/59259055/168481928-f596208b-2515-4220-b032-81699eaf5d43.png)
  最底下的文件，被遮挡了，只显示了前几个

![2](https://user-images.githubusercontent.com/59259055/168481959-00fa39a0-e9e5-4559-9a59-29eaf861c8cc.png)
![3](https://user-images.githubusercontent.com/59259055/168481960-e79c1800-d28b-4a26-ac02-fd5dbd7753ab.png)

切换布局问题，右键点击文件，还要我选择文件！！！
那还得切换回列表布局，然后选择文件，再切回网格布局才能右键

**Comment by anwen-anyi** [comment]
QAQ 哪你直接用列表模式呗...列表模式你可以多选然后在随便一个地方右键它复制的是你选择文件的链接.
或许你需要添加一个 Readme.md文件来给底部填充一下哈哈..
或者自己写个CSS将一部分内容定位在底部..（建议直接用第二行的）
